ext/drpttracer/drpttracer.c

```diff
@@ -577,6 +550,9 @@ drpttracer_end_tracing(IN void *drcontext, INOUT void *tracer_handle,
                (uint8_t *)handle->base + handle->header->data_tail,
                (*output)->sideband_buf_size);
     } else {
+        /* Even only tracing kernel instructions, there are some sideband data. Because we
```

Comment:
"Even when tracing only kernel instructions, there is some sideband data."

Reply:
And in dox too

Reply:
Done.